### ISSUE TITLE
QA: Dollar General

### ISSUE THREAD

**Issue by Sakshi-Handa** [HumanExposure/ChemicalExposure-SSC#1059]
https://ccte-factotum.epa.gov/qa/extractionscript/529/

Complete QA for 300 documents within group for Composition Script Dollar General SDS.

**Comment by cnlutz20** [comment]
@Sakshi-Handa I'm finding that a lot of composition values that are ranges are entered as central composition. ex. 5-10% entered as "central: 5-10" instead of "raw_min: 5 raw_max: 10"

should we just fix these in the data cleaning stage?

**Comment by Sakshi-Handa** [comment]
> @Sakshi-Handa I'm finding that a lot of composition values that are ranges are entered as central composition. ex. 5-10% entered as "central: 5-10" instead of "raw_min: 5 raw_max: 10"
> 
> should we just fix these in the data cleaning stage?

Yes, I think it is okay to report a range in the central field. It might have been bc this group used the generic script. It will be cleaned up in the composition cleaning process.

**Comment by Sakshi-Handa** [comment]
Reopening the Dollar General QA. since we had to redo the extraction. 
The Dollar General generated tables we were extracting from filled in the CAS number for Ethylbenzene (`00-41-4) to every ingredient where no CAS was reported in the SDS. So  in Factotum we had other text and chemicals that were incorrectly curated to Ethylbenzene. 
We decided to redo the extraction, and remove the CAS for Ethylbenzene (100-41-4) where it was incorrectly applied.
